src/sentry/api/endpoints/organization_member_index.py

```diff
@@ -154,25 +164,40 @@ def post(self, request, organization):
 
         serializer = OrganizationMemberSerializer(
             data=request.data,
-            context={"organization": organization, "allowed_roles": allowed_roles},
+            context={
+                "organization": organization,
+                "allowed_roles": allowed_roles,
+                "allow_existing_invite_request": True,
+            },
         )
 
         if not serializer.is_valid():
             return Response(serializer.errors, status=400)
 
         result = serializer.validated_data
 
-        om = OrganizationMember(
-            organization=organization,
-            email=result["email"],
-            role=result["role"],
-            inviter=request.user,
-        )
-
-        if settings.SENTRY_ENABLE_INVITES:
-            om.token = om.generate_token()
-
         with transaction.atomic():
+            try:
+                existing_invite_request = OrganizationMember.objects.get(
```

Comment:
Do we need to even do the lookup first, can we just call delete on this query set